4 - AnimalShelter/Code/FilterTypes.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace AnimalShelter.Code
+{
+    public enum FilterTypes
+    {
+        Fly,
+        Dog,
+        Cat,
+        All
+    }
```

Comment:
Good use of enum for the filters as well, though as a general rule usually your "default" enum value (the first one defined) is either your "all" or "none" value. So I would have had "All" be first.